.travis.yml

```diff
@@ -4,7 +4,7 @@ git:
   depth: 1
 env:
   - GO111MODULE=on # go modules are the default
```

Comment:
We can remove this since it's enabled by default starting 1.13?

Reply:
Done. Thank you for pointing this out!